keywords/ChangesTracker.py

```
@@ -65,9 +64,11 @@ def start(self, timeout=1000, heartbeat=None, request_timeout=None):
         current_seq_num = 0
 
         log_info("[Changes Tracker] Changes Tracker Starting ...")
-
+        start = time.time()
         while not self.cancel:
-
+            if time.time() - start > timeout:
+                logging.error("[Changes Tracker] wait_until: TIMEOUT")
```

raise an exception here rather than logging an error and breaking out